Tasks/DownloadBuildArtifactsV0/DownloadHandlers/DownloadHandlerContainerZip.ts

```diff
@@ -33,12 +33,13 @@ export class DownloadHandlerContainerZip extends DownloadHandler {
      * @returns {Promise<void>} promise that will be resolved once the archive will be unpacked
     */
     private unzipContainer(unzipLocation: string): Promise<void> {
+        const normalizedUnzipLocation = path.normalize(unzipLocation);
```

Comment:
Could it make sense to normailze it earlier? Maybe in https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/DownloadBuildArtifactsV0/main.ts#L82

Reply:
@anatolybolshakov good point, thanks!